CMakeLists.txt

```diff
@@ -56,16 +56,13 @@ find_package(MPI REQUIRED COMPONENTS CXX)
 ########################################################################
 
 find_package(FFTW3 REQUIRED)
-include_directories(${FFTW3_INCLUDES})
 
 ########################################################################
 #Process GROMACS settings
 ########################################################################
 
 if(WITH_XTC)
     find_package(GROMACS REQUIRED)
```

Comment:
We should install this library in our CI and delete our custom find module as gromacs brings it's own now.

Reply:
I.e. `find_package(GROMACS 2021 CONFIG NAMES gromacs gromacs_d)`

Reply:
Yes.

General remark. As far as I can see Gromacs is only used to write `XTC` files. Perhaps it is a little bit of an overkill to require a Gromacs installation just for that...

Reply:
It is optional after all.

Reply:
True. Is Gromacs installed in our buildenv?